composeApp/src/commonMain/kotlin/org/ooni/probe/ui/results/ResultsViewModel.kt

```diff
@@ -127,11 +173,15 @@ class ResultsViewModel(
         val summary: Summary? = null,
         val isLoading: Boolean = true,
         val markAllAsViewedEnabled: Boolean = false,
+        val selectedResults: Set<ResultModel.Id> = emptySet(),
```

Comment:
You could also have wrapped the ResultListItem with `SelectableItem` like we've done in other screens. But you would still need to keep the state inside the ViewModel